### PR TITLE
Add console log format option to CRD

### PR DESCRIPTION
**Pull request by edavidj**
- Adds a log field to the root level of the CRD , which would only allows for the setting of consoleFormat but could later be used to add in other log related fields that the helm chart had
- Added a utils sub-package for liberty specific util customization methods and/or methods that wrap common util methods with additional functionality.
- Sets the default variable value for console format to json, as per the original issue

### CODE REVIEW

File: pkg/utils/utils.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	// validate if it's a new var before appending it to the pod spec
+	for _, v := range targetEnv {
+		if _, found := findEnvVar(v, pts.Spec.Containers[0].Env); !found {
+			pts.Spec.Containers[0].Env = append(pts.Spec.Containers[0].Env, v)
+		}
+	}
+}
+
+// findEnvVars checks if the environment variable is already present
+func findEnvVar(e corev1.EnvVar, envList []corev1.EnvVar) (int, bool) {
```
Comment (arturdzm) [inline pkg/utils/utils.go]: I think its would be better to return &envList[i] (pointer) if found or nil otherwise

File: pkg/utils/utils.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import (
+	openlibertyv1beta1 "github.com/OpenLiberty/open-liberty-operator/pkg/apis/openliberty/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// Utility methods specific to Liberty and it's configuration
+
+// CustomizeEnv adds configured env variables appending configured liberty settings
+func CustomizeEnv(pts *corev1.PodTemplateSpec, la *openlibertyv1beta1.LibertyApplication) {
+	targetEnv := la.GetEnv()
```
Comment (arturdzm) [inline pkg/utils/utils.go]: This should already have been set by CustomizePodSpec and pts object should contain user defined Env already
we only care about appending extra vars to pts

File: pkg/utils/utils.go (hunk 1 above)
Comment (edavidj) [reply pkg/utils/utils.go]: Made the change in recent commit, along with the additional offline suggestions

File: pkg/utils/utils.go (hunk 2 above)
Comment (edavidj) [reply pkg/utils/utils.go]: Good point, got rid of targetEnv entirely and all logic relates to the consoleFormat env var now

**Pull request merged**